package.json

```diff
@@ -32,11 +32,11 @@
     "test:once": "karma start",
     "test:once:ci": "karma start",
     "test": "karma start",
-    "ci": "npm run e2e && npm run test:once:ci",
+    "ci": "npm run e2e:jit && npm run test:once:ci",
```

Comment:
e2e doesn't build the project, which is important in CI, so I added the jit option